explorer/src/components/Consensus/Account/AccountPreviousRewards.tsx

```diff
@@ -1,22 +1,18 @@
 import { TOKEN } from 'constants/general'
-import { Routes } from 'constants/routes'
+// import { Routes } from 'constants/routes'
 import dayjs from 'dayjs'
 import relativeTime from 'dayjs/plugin/relativeTime'
-import {
-  AccountRewards,
-  AllRewardForAccountByIdQuery,
-  AllRewardForAccountByIdQueryVariables,
-} from 'gql/graphql'
-import { useSquidQuery } from 'hooks/useSquidQuery'
-import { useWindowFocus } from 'hooks/useWindowFocus'
+// import { AllRewardForAccountByIdQuery, AllRewardForAccountByIdQueryVariables } from 'gql/graphql'
+// import { useSquidQuery } from 'hooks/useSquidQuery'
+// import { useWindowFocus } from 'hooks/useWindowFocus'
 import { useParams } from 'next/navigation'
-import { FC, useCallback, useEffect, useMemo, useState } from 'react'
+import { FC, useCallback, useEffect, useState } from 'react' // , useMemo
```

Comment:
This comment?

Reply:
Thanks, yes I had not clean that page, I just did
I'm not sure if this components still make sense now that we are about to release mainnet, if it does, then I'll need to update the logic to use the same logic as testnet reward section